x/concentrated-liquidity/model/pool_test.go

```diff
@@ -367,12 +367,22 @@ func (s *ConcentratedPoolTestSuite) TestApplySwap() {
 			expectErr:        nil,
 		},
 		{
-			name:             "new tick is equal to min tick",
+			name:             "new tick is equal to min initialized tick",
 			currentLiquidity: DefaultLiquidityAmt,
 			currentTick:      DefaultCurrTick,
 			currentSqrtPrice: DefaultCurrSqrtPrice,
 			newLiquidity:     DefaultLiquidityAmt,
-			newTick:          types.MinTick,
+			newTick:          types.MinInitializedTick,
+			newSqrtPrice:     DefaultCurrSqrtPrice,
+			expectErr:        nil,
```

Comment:
A little confused on this one. If the newTick is going to be minInitializedTick, wouldn't that make the newCurrentTick =  minInitializedTick? And isn't this not allowed by the ApplySwap check here? https://github.com/osmosis-labs/osmosis/pull/5581/files#diff-9ff53a5917532eb5c22f23f3718d45ff3b7d738df6f17652b3af5b39a30d583eR290

Reply:
The line you linked disallows "current tick" from being smaller than `MinCurrentTick`. `MinCurrentTick` is smaller than `MinInitializedTick` (difference of 1).

So it is allowed for `MinInitializedTick` to be applied

Reply:
This makes sense, thanks